samples/nrf9160/aws_fota/src/main.c

```diff
@@ -39,7 +39,7 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_LTE_AUTO_INIT_AND_CONNECT),
 /* Define the length of the IMEI AT COMMAND response buffer */
 #define CGSN_RESP_LEN 19
 #define IMEI_LEN 15
-#define CLIENT_ID_LEN (IMEI_LEN + sizeof("nrf-"))
+#define CLIENT_ID_LEN (CONFIG_NRFCLOUD_CLIENT_ID_PREFIX_LEN + IMEI_LEN)
```

Comment:
This sample does not use nRF Cloud by default, so this config could only be used if nRF Cloud is enabled (provided it's moved to nRF Clouds' Kconfig).

Reply:
Good point, will move it to the library level.